### DISCUSSION TITLE
Azure OpenAI application requires a company email address.

### DISCUSSION THREAD

**Discussion by vyrotek** [microsoft/hack-together-dotnet#14] [General]
According to the Azure OpenAI application here, we will be denied if we use a personal email. https://aka.ms/oaiapply 

> Your Company Email Address - Applications submitted with a personal email address (e.g. [gmail.com](http://gmail.com/), [hotmail.com](http://hotmail.com/), [outlook.com](http://outlook.com/), etc.) will be DENIED.

Any possibility of relaxing this for the hackathon so we can use the Azure OpenAI APIs on our hackathon projects but not have them associated to our work accounts? I imagine many of us already have personal Azure accounts we'd like to use. This seems very limiting.

**Comment by codemillmatt** [comment]
Hi @vyrotek (again 😄) ... I have some good news. We're working on a way that will give participants limited access (capped to a reasonable dollar amount per person) to Azure OpenAI. I'll have more complete details on this soon & what to do in order to get it.

**Comment by aaronpowell** [comment]
I've just created an announcement on how you can get access - https://github.com/microsoft/hack-together-dotnet/discussions/23

Good luck!
